test/functional/autoupdate/aup-basic.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
```

Comment:
Just run this test on if TRAVIS == true because we write files in /usr and we don't want to do it in dev machines.

Reply:
Okay